examples/pycaffe/layers/pascal_multilabel_datalayers.py

```diff
@@ -0,0 +1,262 @@
+# imports
+import json, time, pickle, scipy.misc, skimage.io, caffe
+
+import numpy as np
+import os.path as osp
+
+from xml.dom import minidom
+from random import shuffle
+from threading import Thread
+from PIL import Image
+
+from pascal_multilabel_with_datalayer_tools import SimpleTransformer
```

Comment:
Thanks a ton Evan, very nicely done. I'm not sure the new layer code is ideal from a tutorial perspective though. While certainly more professional, I think it obscures things a bit, and the reader have to jump back and forth to follow the line of execution. I tend to favour showing people the easiest, dumbest, and most straight-forward way of doing things, and let them take it from there. Does that makes sense? What do other folks think? @shelhamer?